_layouts/ranking.html

```diff
@@ -0,0 +1,252 @@
+---
+layout: default
+subpage-title: Ranking
+---
+<script src="https://unpkg.com/papaparse@5.3.0/papaparse.min.js" ></script>
+<script>
+  let rankingCategory = 'ord-percentage';
+  let cities = [];
+
+  const selectRanking = (element) => {
+    document.getElementById(rankingCategory).classList.remove('ranking-select__button--active');
+    element.classList.add('ranking-select__button--active');
+    rankingCategory = element.id;
+    const listRoot = document.getElementById('city-list');
+    listRoot.innerHTML = '';
+    if (rankingCategory === 'ord-new') {
+      cities.sort((a,b) => parseInt(b.added_total) - parseInt(a.added_total));
+    } else {
+      cities.sort((a,b) => b.perc_total - a.perc_total);
+    }
+    cities.forEach((city, index) => {
+      listRoot.innerHTML =  listRoot.innerHTML + renderCityElement(city, cities[0], index);
+    });
+  }
+
+  const handleElementClick = (index) => {
+    const shownElements = [...document.getElementsByClassName('shown')].forEach((element) => {
+      element.classList.add('hidden');
+      element.classList.remove('shown');
+    });
+    elementToShow = document.getElementById(`details-${index}`)
+    elementToShow.classList.remove('hidden');
+    elementToShow.classList.add('shown');
+  }
+
+  const showMap = (cityName) => {
+    window.location = `https://api.cargorocket.de/dr/docs/datenrennen_maps/${cityName}_map.html`;
+  }
+
+  const renderContribute = () => {
+    return `
+        <div class="contribution__modal">
+          <h3 class="contribution__modal__title">Werde Datenspender!</h3>
+          <p class="contribution__modal__description">
+            Unterstütze deine Region beim Lastenradrouting mit OpenData.<br>
+            Das geht ganz einfach! Rufe einen unserer Datenspender Links auf und unterstütze deine Gemeinde mit hilfreichen Informationen für das Lastenradrouting.<br>
```

Comment:
It's actually "Landkreis" not "Gemeinde" (there are 1101 Gemeinden in Ba-Wü)

Reply:
To be consistent on our website, and for better readability I would use "Lastenrad-Routing" instead of "Lastenradrouting"